external/mve11/mve_run.m

```diff
@@ -1,286 +0,0 @@
-function [x,E] = mve_run(A,b,x0)
```

Comment:
I was worried that it would get called by accident. The original zip file has two files with the name mve_run.m, one with functions in different files and one with all functions in 1 file. I updated the version with functions split across files, and deleted this one so that it would not get called by accident.